### PR TITLE
Bi-Enc Jaseci_kit pipeline crash fix

### PR DESCRIPTION
**Pull request by AshishMahendra**
## Describe your changes
Disabled test cases that perform training 
## Link to related issue
N/A
## Does this introduce new feature or change existing feature of Jaseci? If so, please add tests.
N/A
## Will this impact Jaseci users? If so, please write a paragraph describing the impact.
N/A
## Anything in this PR should the Jaseci developer/contributor community pay particular attention to? If so, please describe.
N/A

### CODE REVIEW

**Comment by ypkang** [comment]
@AshishMahendra this ready for review and merge?

**Comment by AshishMahendra** [comment]
> @AshishMahendra this ready for review and merge?

Yes, I have skipped the training function, it works fine now

File: jaseci_kit/jaseci_kit/modules/text_seg/text_seg.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def get_segments(text: str, threshold: float = 0.7):
 
 
 @jaseci_action(act_group=["text_seg"], allow_remote=True)
-def load_model(model_name: str):  # modelname could be ("wiki", "legal")
+def seg_load_model(model_name: str):  # modelname could be ("wiki", "legal")
```
Comment (ypkang) [inline jaseci_kit/jaseci_kit/modules/text_seg/text_seg.py:72]: Why does this need to be changed?

**Comment by ypkang** [comment]
@AshishMahendra this looks mostly fine except for one question I had. look at inline comments.

Also, I noticed this PR has a lot of old commits. I think I mentioned this before, but please create a new branch from your fork's main everytime for a new PR, instead of using an existing branch. It is not good practice.

File: jaseci_kit/jaseci_kit/modules/text_seg/text_seg.py (hunk 1 above)
Comment (AshishMahendra) [reply jaseci_kit/jaseci_kit/modules/text_seg/text_seg.py:72]: As discussed, this is for parallel execution issue, we may have to find a better solution for this

**Comment by ypkang** [comment]
@AshishMahendra I made some updates to this branch and going to merge it. Just FYI.

**Review by ypkang — APPROVED** [review]

**Pull request merged**